conf/live_2.routes

```diff
@@ -0,0 +1,10 @@
+# API 2.0 routes
+
+# routes published to WSO2 because included in the RAML definition (returning 200 OK)
+GET        /                                   uk.gov.hmrc.apiplatformtest.controllers.CiaoController.handleEmpty()
+GET        /ciao                               uk.gov.hmrc.apiplatformtest.controllers.CiaoController.handleCiao(surname: Option[String], firstName: Option[String], middleName: Option[String])
```

Comment:
I just tested this route locally.
It does not work as expected apparently.
Only the first query parameter is recognised.
You can reproduce the issue with these steps:
1) run the microservice locally with `sbt run`
2) make this call `curl -v -X GET http://localhost:9000/ciao?firstName=AAA&surname=BBB&middleName=CCC`
3) the JSON response will display only the first query parameter (`firstName` in this case, but you can try also changing the order of the parameters if you want)

Reply:
This works fine for me when using Postman to call the API:
```
http://localhost:9000/ciao?firstName=Bob&surname=Builder&middleName=The
```
response:
```
{
    "uri": "/ciao?firstName=Bob&surname=Builder&middleName=The",
    "resourceDetails": "Ciao: Some(Builder), Some(Bob), Some(The)"
}
```

Reply:
& is interpreted by the shell - try quoting the whole URI in single quotes

Reply:
Thanks.
If I wrap the URL with single quotes in my shell curl request, it works also for me now.
```
curl -v -X GET 'http://localhost:9000/ciao?firstName=AAA&surname=BBB&middleName=CCC'
```